ci/jenkins/README.md

```diff
@@ -93,60 +93,62 @@ DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase windows-e2e --registry ${DOCKER_REGISTRY}
 ```
 
-* windows conformance: community tests on Windows cluster with Docker runtime, focusing on "Conformance|sig-windows" and
-  "sig-network", and skipping "LinuxOnly", "Slow", "Serial", "Disruptive", "Flaky", "Feature", "sig-cli", "sig-storage",
-  "sig-auth", "sig-api-machinery", "sig-apps", "sig-node", "Privileged", "should be able to change the type from",
-  "[sig-network] Services should be able to create a functioning NodePort service [Conformance]", "Service endpoints
-  latency should not be very high".
+* windows containerd e2e: e2e tests in a Windows setup with Containerd runtime.
```

Comment:
refer #5630 ,windows container runtime is defaulted to Containerd, prefer to remove the term containerd runtime from documentations and trigger phrases to keep the trigger phase short